nilearn/datasets/utils.py

```diff
@@ -141,9 +146,9 @@ def _chunk_read_(response, local_file, chunk_size=8192, report_hook=None,
             total_size = response.info().get('Content-Length').strip()
         total_size = int(total_size) + initial_size
     except Exception as e:
-        if verbose > 1:
+        if verbose > 2:
```

Comment:
This warning (which I don't think is really necessary; there's no problem here that I can see) was at the same verbosity level as printing the querystring on the URL.